docs/strategies/fallback.md

```diff
@@ -65,7 +65,50 @@ new ResiliencePipelineBuilder<UserAvatar>()
 | `FallbackAction` | `Null`, **Required**                                                       | Fallback action to be executed.                                                             |
 | `OnFallback`     | `null`                                                                     | Event that is raised when fallback happens.                                                 |
 
-## Patterns and anti-patterns
+## Patterns
+
+### Fallback after retries
+
+When designing resilient systems, a common pattern is to use a fallback after multiple retry attempts. This approach is especially relevant when a fallback strategy can provide a sensible default value.
+
+<!-- snippet: fallback-after-retries -->
+```cs
+var pipeline = new ResiliencePipelineBuilder<HttpResponseMessage>()
+    .AddFallback(new()
+    {
+        ShouldHandle = new PredicateBuilder<HttpResponseMessage>()
```

Comment:
In order to empathize that  the fallback would trigger after the retry attempts exhausted we could reuse the `PredicateBuilder` between the two strategies.

Reply:
I like this suggestion. It was something I was considering doing when I started migrating to v8 but didn't dare to do because I wasn't sure it was allowed (thinking maybe the builder would have some state or whatever..). Having it here would remove such doubts.

And as suggested, it makes it clear to the reader of the code that they handle the same problem.

Reply:
The only "state" of the `PredicateBuilder` is the [list of predicates](https://github.com/App-vNext/Polly/blob/main/src/Polly.Core/PredicateBuilder.TResult.cs#L9).

Reply:
>In order to empathize that the fallback would trigger after the retry attempts exhausted we could reuse the PredicateBuilder between the two strategies.
 
Good suggestion! Done.